src/patterns/start-pages/index.md.njk

```diff
@@ -1,41 +1,72 @@
+- let users sign in, resume an application they’ve already started or update their details (if relevant)
+- include any other information that most users are likely to need before they start using the service online - for example, how much it costs to use the service and roughly how long it will take
+- include details of other ways to access the service — for example, by telephone or by completing a paper form
 
-You can use the coded example here to test what works best for your users in prototypes.
+You’ll also need to list any documents or information the user will need to help them complete the service (there’s no need to list information that a user is likely to know from memory — for example, their own date of birth).
 
-## How it works
+You can list the documents or information at the start point, if it doesn’t make the page too complicated. Or you can move these details to a separate ‘What you’ll need’ page, served within the service subdomain.
+
+## Eligibility
+
+Avoid using the start point to provide complex eligibility information — instead, ask questions inside the service to establish whether the user is eligible.
+
+Read [guidance on designing how GOV.UK content and transactions work together](https://www.gov.uk/service-manual/design/govuk-content-transactions).
+
+## Choosing a format for a start point on GOV.UK
+
+The options available depend on whether the start point will be published as ‘mainstream’ or ‘Whitehall’ content.
+
+The content team at GDS is responsible for creating mainstream start points (as with other mainstream content). ‘Mainstream’ content is content aimed at the general public — or at meeting business needs that aren’t sector-specific.
+
+### Mainstream start points
+
+For a mainstream start point, the options are:
+- a simple start page
+- a start point within a multipart guide
+
+Use a simple start page if users can start using the service without needing much context.
+
+<img src="simple-start-page.png" alt="A screenshot showing an example of a simple start page, taken from the coronavirus shielding support service. The page includes a heading, a few simple paragraphs and bullet points to explain who can use the service, and a start button labelled 'Register or update your details'.">
+
+If the start point is linked to a wider process that needs more explanation, create an ‘Apply’ section within a multipart guide. 
 
-Start pages include 4 main elements:
+For example, before a user applies for probate (the right to manage someone’s estate after they die) they need to understand whether probate is required, whether they’re the right person to apply, how the details of the will and the tax situation affect what they’re supposed to do, how to get the relevant information together and how the different outcomes affect what they'll need to do next.
```

Comment:
Long sentence. Could bullet.

```suggestion
For example, before a user applies for probate (the right to manage someone’s estate after they die), they need to:

- understand whether probate is required
- whether they’re the right person to apply
- how the details of the will and the tax situation affect what they’re supposed to do
- how to get the relevant information together
- how the different outcomes affect what they'll need to do next

```

Reply:
@StephenGill and I [discussed this one in the Word doc](https://docs.google.com/document/d/1tqT131mXKmpreYkrpe4OmUc03RCUr-tuKLN7T4CMWnw/edit?disco=AAAAWEvT3Zc). 

Given that the purpose of this paragraph is to show how complicated the process is (and not the information itself), the bullet points would distract from more key info on this page...

Reply:
Fair enough. I just wondered at the length - 67 words - when the [Service Manual](https://www.gov.uk/guidance/content-design/writing-for-gov-uk#how-people-read) (see 'Short Sentences') recommends 25 at most. Seems like a lot to make users read.

Could always delete a couple of clauses, and replace them with "and so on", to get the word count down.

Reply:
Could we perhaps pick 3 to get the point across @StephenGill?

```suggestion
For example, before a user applies for probate (the right to manage someone’s estate after they die) they need to understand whether probate is required, whether they’re the right person to apply, how the details of the will and the tax situation affect what they’re supposed to do, and so on.
```